transport-security.md

```diff
@@ -123,7 +121,7 @@ Services (potentially) including script code (e.g. JavaScript) in their response
 - Ensure sending intended content type headers in your response matching your body content e.g. `application/json` and not `application/javascript`.
 
 ### HTTP Return Code
-HTTP defines status codes. When designing a REST API, don't just use `200` for success or `404` for error. Always use the semantically appropriate [status code](https://tools.ietf.org/html/rfc7231#section-6) for the response.
+HTTP defines status codes. When designing a REST API, don't just use `200` for success or `404` for error. Always use the semantically appropriate <a href="https://www.rfc-editor.org/rfc/rfc9110#name-status-codes">status code</a> [[[rfc9110]]] for the response.
```

Comment:
Again render versie mismatch waarschijnlijk maar ik zie nog een link naar [HTTP Semantics](https://httpwg.org/specs/rfc9110.html)
![image](https://github.com/user-attachments/assets/e76a58d8-1b60-4611-b387-d0a93198d0e6)

Reply:
Dit moet idd maar 2 brackets zijn ipv 3, omdat we niet de naam willen overnemen omdat dat al in de link staat.

Reply:
Ja, om er voor te zorgen dat hij correct als normatieve referentie erin komt. Anders linken we en is het niet normatief.